src/components/RadioButton/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./styles.scss";
+
+interface RadioButtonProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  id: string;
+  label: string | React.ReactNode;
+}
+
+export const RadioButton = ({ id, label, ...props }: RadioButtonProps) => (
+  <div className="RadioButton">
+    <input type="radio" id={id} {...props} />
```

Comment:
❓ Will `{...props}` render with the correct HTML tag syntax?

Reply:
Yes, all the native input props will render correctly (name, value, etc).